tests/pc/test_lag_member.py

```diff
@@ -0,0 +1,360 @@
+    """
+    Restore ptf configuration
+
+    Args:
+        ptfhost: PTF host object
+        ptf_lag_map: imformation about lag in ptf
+    """
+    ptfhost.shell("ip link set {} nomaster".format(PTF_LAG_NAME))
+
+    for ptf_lag_member in ptf_lag_map[PTF_LAG_NAME]["port_list"]:
+        ptfhost.shell("ip link set {} nomaster".format(ptf_lag_member))
+        ptfhost.shell("ip link set {} up".format(ptf_lag_member))
+        
+    ptfhost.shell("ip link del {}".format(PTF_LAG_NAME))
+    ptfhost.shell("ip addr del {} dev {}".format(ptf_lag_map[ATTR_PORT_NOT_BEHIND_LAG]["ip"], ptf_lag_map[ATTR_PORT_NOT_BEHIND_LAG]["port_name"]))
+    restart_ptf_nn_agent(ptfhost)
+
+def setup_dut_lag(duthost, dut_ports, vlan):
+    """
+    Setup dut lag
+
+    Args:
+        duthost: DUT host object
+        dut_ports: ports need to configure
+        vlan: information about vlan configuration
+
+    Returns:
+        information about dut lag
+    """
+    duthost.shell("config acl remove table EVERFLOW")
+    duthost.shell("config acl remove table EVERFLOWV6")
+    duthost.shell("config portchannel add {}".format(DUT_LAG_NAME))
+
+    lag_port_list = []
+    port_list_idx = 0
+    port_list = list(dut_ports[ATTR_PORT_BEHIND_LAG].values())
+    for _ in range(1, 10000):
+        port_name = port_list[port_list_idx]
+        remove_ip_from_port(duthost, port_name)
+        remove_member_from_vlan(duthost, "1000", port_name)
+        duthost.shell("config portchannel member add {} {}".format(DUT_LAG_NAME, port_name))
+        lag_port_list.append(port_name)
+        port_list_idx += 1
+
+        if len(lag_port_list) == len(dut_ports[ATTR_PORT_BEHIND_LAG]):
+            break
+
+    duthost.shell("config vlan add {}".format(vlan["id"]))
+    duthost.shell("config interface ip add Vlan{} {}".format(vlan["id"], vlan["ip"]))
+    add_member_to_vlan(duthost, vlan["id"], DUT_LAG_NAME, "-u")
+    remove_member_from_vlan(duthost, 1000, dut_ports[ATTR_PORT_NOT_BEHIND_LAG]["port_name"])
+    remove_ip_from_port(duthost, dut_ports[ATTR_PORT_NOT_BEHIND_LAG]["port_name"])
+    add_member_to_vlan(duthost, vlan["id"], dut_ports[ATTR_PORT_NOT_BEHIND_LAG]["port_name"], "-u")
+
+    lag_port_map = {}
+    lag_port_map[DUT_LAG_NAME] = lag_port_list
+    lag_port_map[ATTR_PORT_NOT_BEHIND_LAG] = dut_ports[ATTR_PORT_NOT_BEHIND_LAG]
+    lag_port_map["vlan"] = vlan
+    return lag_port_map
+
+def setup_ptf_lag(ptfhost, ptf_ports, vlan):
+    """
+    Setup ptf lag
+
+    Args:
+        ptfhost: PTF host object
+        ptf_ports: ports need to configure
+        vlan: information about vlan configuration
+
+    Returns:
+        information about ptf lag
+    """
+    ip_prefix = ".".join(vlan["ip"].split("/")[0].split(".")[0:3])
+    mask_len = vlan["ip"].split("/")[1]
+    lag_ip = "{}.2/{}".format(ip_prefix, mask_len)
+    port_not_behind_lag_ip = "{}.3/{}".format(ip_prefix, mask_len)
+
+    ptfhost.shell("ip link add {} type bond".format(PTF_LAG_NAME))
```

Comment:
Can we extract these command as a method of ptfhost.py? I feel they can be very helpful.
```
def create_bond(self, bond_name, bond_ip)
def startup_bond(self, bond_name) # or startup_intf?
def add_port_to_bond(self, bond_name, port_name)
def remove_port_from_bind(self, bond_name, port_name)
```

Reply:
Done